### DISCUSSION TITLE
Markdown files still showing as text in Quicklook

### DISCUSSION THREAD

**Discussion by ajcullum** [sbarex/QLMarkdown#56] [Q&A] [answered]
I had used QLMarkdown on a previous MacOS install and it worked perfectly. But under a new install of the current Big Sur (11.1) I'm having no luck. I can view Markdown files just fine within QLMarkdown, but Quicklook still shows a *.md file as plain text.  So two questions to start that I hope will help troubleshoot:

* Am I correct that all I should have to do is run a copy of QLMarkdown to active the Quicklook add-on? No need to install Discount or any other Markdown interpreter?

* If not, where should I look for files that QLMarkdown should have installed? I want to make sure the setup happened like it was supposed to.

Thanks for any info!

**Comment by sbarex** [comment]
You don't need a markdown interpreter, the application has everything you need.

Check in System preferences, Extensions, Quicklook, if the `QLMarkdown Preview extension` is enabled.
If it does not appear in the list, close the preferences. Put the application in the trash and then put it back in the Applications folder and open it. It should then appear within the System preferences.

**Reply by ajcullum** [reply]
Thanks for the reply. The Quick Look extension is active. I tried the Trash-Applications round-trip, but no luck there.

To see if there was some deeper MacOS problem, I tried the app PreviewMarkdown that's available through the App Store. That did work, but it doesn't support tables and the rendering is ugly, so I'd still like to get QLMarkdown working!. 

So I guess I'm back to the question of whether there's something I can look for - a file, or a log entry - to tell me what might be missing. 

Thanks again!

<img width="672" alt="Screen Shot 2021-02-05 at 11 24 45 AM" src="https://user-images.githubusercontent.com/36862784/107067568-11278700-67a5-11eb-8667-1f78f1fa11a5.png">

**Comment by setanarut** [comment]
@ajcullum There may be conflict with the _Bear Quicklook Extension_. Also There should be no other markdown plugins in the `~/Library/QuickLook` or `/Library/QuickLook`

**Reply by ajcullum** [reply]
Thanks. I tried disabling the Bear extension with no luck. And the `/Library/QuickLook` folder is empty, and `~/Library/QuickLook` doesn't exist.

**Comment by sbarex** [comment]
In the System preferences, I believe it is possible to drag the various extensions into the preferred order of priority. On the other hand, it is not possible to establish a priority for the old qlpreview generators present in the library/quicklook folder.

**Reply by ajcullum** [reply]
That tip on reordering is good - thanks for that!

**Comment by ajcullum** [comment, marked as answer]
Downloaded the latest update (1.0b23) today, and it went from not working to working! Not sure what the code change was that did it, but all is good! Thanks!

**Comment by punkyard** [comment]
hi
i've installed QLM for the first time, on Monterey
the extension was not in the library after first launch so copied it manually
i put MQL in the trash, get it out, but it only shows raw text (which already better)
I can't see QuickLook in System Prefs

could one share one more thing I should try to have a preview of .md files in quicklook?
thanks if you can ;)

thanks for this app, hopefully it will work for me too

**Comment by punkyard** [comment]
here is a view on my prefs - which should be default I think
![Capture d’écran 2024-09-01 à 19 46 54](https://github.com/user-attachments/assets/9c73089e-15f5-4d88-8d74-6f675efe676a)

**Comment by sbarex** [comment]
Make sure the Quick Look extension is enabled (in system preferences, I don't remember where it is in Monterey, in the extensions / Quick Look section)
